package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "mock-inspect",
-    "version": "4.0.2",
+    "version": "5.0.0",
```

Comment:
Please could you revert this change and instead run `npm version major` as this will update the changelog. In addition, it will show up as a separate commit to just bump the version.

Reply:
I somehow missed this; I will do this later.